src/asmcnc/skavaUI/screen_door.py

```diff
@@ -155,7 +154,7 @@
                     size: self.parent.size
                     x: self.parent.x + 80
                     y: self.parent.y
-                    opacity: 0
+                    opacity: 1
```

Comment:
The text "Preparing to resume, please wait" is displayed on immediate entry to the screen. Currently it is only shown the second time you enter the screen, because opacity is set to 1 when the ready_to_resume function is called (and is never set back to 0). So I assumed this was unintentional and set it to 1 so that the text is displayed on first entry.